content/ko/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,347 @@
+NAME             READY   STATUS    RESTARTS   AGE
+frontend-pxj4r   1/1     Running   0          5s
+pod1             1/1     Running   0          13s
+pod2             1/1     Running   0          13s
+```
+
+이러한 방식으로 레플리카셋은 템플릿을 사용하지 않는 파드를 소유하게 된다.
+
+## 레플리카셋 매니페스트 작성하기
+
+레플리카셋은 모든 쿠버네티스 API 오브젝트와 마찬가지로 `apiVersion`, `kind`, `metadata` 필드가 필요하다.
+레플리카셋은 항상 레플리카셋이다.
+쿠버네티스 1.9에서의 레플리카셋의 버전은 API 버전 `apps/v1`과 동일하고 기본적으로 활성화 되어있다. API 버전 `apps/v1beta2`은 더 이상 사용하지 않는다.
+API버전에 대해서는 `frontend.yaml` 예제의 첫 번째 줄을 참고한다.
+
+레플리카셋도 [`.spec` 섹션](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status)이 필요하다.
+
+### 파드 템플릿
+
+`.spec.template`은 레이블을 붙이도록 되어있는 [파드 템플릿](/docs/concepts/workloads/Pods/pod-overview/#pod-templates)이다.
+우리는 `frontend.yaml` 예제에서 `tier: frontend`이라는 레이블을 하나 가지고 있다.
+이 파드를 다른 컨트롤러가 취하지 않도록 다른 컨트롤러의 셀렉터와 겹치지 않도록 주의해야 한다.
+
+템플릿의 [재시작 정책](/docs/concepts/workloads/Pods/pod-lifecycle/#restart-policy) 필드인 
+`.spec.template.spec.restartPolicy`는 기본값인 `Always`만 허용된다.
+
+### 파드 셀렉터
+
+`.spec.selector` 필드는 [레이블 셀렉터](/docs/concepts/overview/working-with-objects/labels/)이다.
+[앞서](#레플리카-셋의-작동-방식) 논의한 것처럼 이 레이블은 파드를 식별하는데 사용된다.
+우리 `frontend.yaml` 예제에서의 셀렉터는 다음과 같다.
+```shell
+matchLabels:
+	tier: frontend
+```
+
+레플리카셋에서 `.spec.template.metadata.labels`는 `spec.selector`과 일치해야 하며 
+그렇지 않으면 API에 의해 거부된다.
+
+{{< note >}}
+2개의 레플리카셋이 동일한 `.spec.selector`필드를 지정하지만 각 필드가 `.spec.template.metadata.labels`과 `.spec.template.spec`인경우 각 레플리카 셋은 다른 레플리카 셋이 생성한 파드를 무시한다.
+{{< /note >}}
+
+### 레플리카
+
+`.spec.replicas`를 설정해서 동시에 실행할 파드의 수를 지정할 수 있다.
+레플리카셋은 파드의 수가 일치하도록 생성 및 삭제한다.
+
+만약 `.spec.replicas`를 지정하지 않으면 기본값은 1이다.
+
+## 레플리카셋 작업
+
+### 레플리카셋과 해당 파드 삭제
+
+레플리카셋 및 모든 파드를 삭제하려면 [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete)를 사용한다. [가비지 수집기](/docs/concepts/workloads/controllers/garbage-collection/)는 기본적으로 종속되어있는 모든 파드를 자동으로 삭제한다.
+
+REST API또는 `client-go` 라이브러리를 이용할 때는 -d 옵션으로 `propagationPolicy`를 `Background`또는 `Foreground`로 
+설정해야 한다.
+예시:
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/extensions/v1beta1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
+> -H "Content-Type: application/json"
+```
+
+### 레플리카셋만 삭제하기
+
+레플리카셋을 `--cascade=false` 옵션과 함께 [`kubectl delete`](/docs/reference/generated/kubectl/kubectl-commands#delete)를 사용하면 연관 파드에 영향을 주지 않고 삭제할 수 있다.
+REST API 또는 `client-go` 라이브러리를 이용할 때는 `propagationPolicy`에 `Orphan`을 설정해야 한다.
+예시:
+```shell
+kubectl proxy --port=8080
+curl -X DELETE  'localhost:8080/apis/extensions/v1beta1/namespaces/default/replicasets/frontend' \
+> -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Orphan"}' \
+> -H "Content-Type: application/json"
+```
+
+원본이 삭제되면 새 레플리카셋을 생성해서 대체할 수 있다.
+기존 `.spec.selector`와 신규 `.spec.selector`가 같으면 새 레플리카셋은 기존 파드를 선택한다.
+하지만 새 레플리카셋은 기존 파드에 대해 새로운 파드 템플릿 또는 다른 파드 템플릿과 일치하도록 활동하지 않는다.
```

Comment:
[원문]However, it will not make any effort to make existing Pods match a new, different pod template.
원문을 최대한 반영하여 수정해 보았습니다. 
```suggestion
하지만 신규 레플리카셋은 기존 파드를 신규 레플리카셋의 새롭고 다른 파드 템플릿에 일치시키는 작업을 수행하지는 않는다.
```